### PR TITLE
Código Hub Readme

### PR DESCRIPTION
**Pull request by mahbubiftekhar**
Read me so far

### CODE REVIEW

**Comment by ZeroSum24** [comment]
I've got this one lads 👍

File: README.md [hunk 1]
```diff
@@ -1,11 +1,65 @@
...
 
+## Project Overview
 
-We're building a Dapp using React, tied in with 3box (for user management and data storage), deployed on Fleek and integrated with both Ethereum (for user incentivisation) and Filecoin (to manage distributed firmware storage).
+[Código](https://codigo-hub.com/) is a firmware distribution service built on top of the Ethereum blockchain and IPFS. Código natively allows developers to upload firmware for different devices and provides rudimentary search capabilities for users. Users can download firmware either by requesting a specific firmware image from a known developer or utilising a built-in web of trust model to automatically select a trusted developer.
+
+A full
```
Comment (Alex-Shand) [inline README.md]: This probably shouldn't be here

File: README.md [hunk 2]
```diff
@@ -1,11 +1,65 @@
...
+[Código Hub](https://codigo-hub.com/)  is built using React.js, Redex, Redux Persist and 3box. Intergration with several custom Ethereum 1.0 smart contracts is provided by Metamask. Filecoin and IPFS integration is provided by the Textile Powergate Javascript library and a hosted Powergate instance provided by Textile. Websocket MQTT is used to communicate with and automatically deploy firmware to compatible devices.
+</div>
+
+## System Design Diagram
+<img alt="System Diagram" src="src/images/overview.png">
+
+
+
+## [Demo Video](https://codigo-hub.com/)
+You can view our demo video by clicking [here](https://codigo-hub.com/).
```
Comment (Alex-Shand) [inline README.md:33]: Presumably this is waiting for the youtube URL

File: README.md [hunk 3]
```diff
@@ -1,11 +1,65 @@
...
+
+You can also try the site by clicking [here](https://codigo-hub.com/)
+
+## Technologies Used
+[React](https://reactjs.org/) <br />
+[Fleek](https://fleek.co/) <br />
+[Textile](https://codigo-hub.com/) <br />
```
Comment (Alex-Shand) [inline README.md]: https://textile.io/

File: README.md [hunk 4]
```diff
@@ -1,11 +1,65 @@
...
+
+## [Demo Video](https://codigo-hub.com/)
+You can view our demo video by clicking [here](https://codigo-hub.com/).
+
+You can also try the site by clicking [here](https://codigo-hub.com/)
+
+## Technologies Used
+* [React](https://reactjs.org/) <br />
+* [Fleek](https://fleek.co/) <br />
+* [Textile](https://codigo-hub.com/) <br />
```
Comment (Alex-Shand) [inline README.md:40]: https://textile.io/

Comment (mahbubiftekhar) [reply README.md:40]: Thanks, changed

**Comment by ZeroSum24** [comment]
@mahbubiftekhar @Alex-Shand why "moving instead to a system like fixed membership numbers" in the last item? Never heard of this before

**Pull request merged**